src/client/src/orangehrmPimPlugin/components/EditSalaryComponent.vue

```diff
@@ -64,7 +64,7 @@
           <oxd-grid-item>
             <oxd-input-field
               v-model="salaryComponent.salaryAmount"
-              :label="$t('pim.amount')"
+              :label="$t('claim.amount')"
```

Comment:
shall we move this to `general.yaml` instead? since it's being used in multiple places

Reply:
This will be problematic when filter based on module in language customization screen

Reply:
added to general.yaml and removed from pim and claim